articles/logic-apps/logic-apps-deploy-azure-resource-manager-templates.md

```diff
@@ -54,7 +54,7 @@ To deploy a logic app template from an Azure Resource Group project that you cre
 
 ## Deploy with Azure PowerShell
 
-To deploy to a specific Azure resource group, use this command:
+To deploy to a specific *Azure resource group*, the following command can be used:
```

Comment:
Hi @whaakman, thanks for your contribution! Please change this edit to active voice:

"...use the following command:"